pkg/api/v1/types.go

```diff
@@ -2251,6 +2259,9 @@ type Event struct {
 
 	// The number of times this event has occurred.
 	Count int `json:"count,omitempty"`
+
+	// Type of this event (Normal, Warning), new types could be added in the future
+	Type string `json:"type,omitempty"`
```

Comment:
I see your point about severity levels. I agree external filtering/alerting is the way to go, and is in fact what I wanted events to go into all along, but no time :(

I tend to think that it'd be better without type at all, considering the above. I don't see how local determinations of normal vs warn can be reliably made.